app/src/main/java/com/github/se/cyrcle/ui/map/MapScreen.kt

```diff
@@ -66,32 +69,28 @@ const val LAYER_ID = "0128"
 fun MapScreen(
     navigationActions: NavigationActions,
     parkingViewModel: ParkingViewModel,
-    state: MutableState<Double> = remember { mutableDoubleStateOf(defaultZoom) }
```

Comment:
I think the Ci doesn't work because you remove state which is an value created for the tests.

Reply:
yeah yeah the PR is not marked as ready for review, this is fixed in the final version

Reply:
It is ready now, you can actually start reviewing